perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/TasksManagerMethod.java

```diff
@@ -106,15 +106,41 @@ public Integer call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Return list of only newest TaskResults by a Task for GUI.
+	 *
+	 * @param task int Task
+	 * @return List<TaskResult> Results
+	 */
+	getTaskResultsByTaskOnlyNewest {
+		public List<TaskResult> call(ApiCaller ac, Deserializer parms) throws PerunException {
+			return ac.getTasksManager().getTaskResultsByTaskOnlyNewest(ac.getSession(), parms.readInt("task"));
+		}
+	},
+
+	/*#
+	 * Return list of TaskResults by a Task and destination for GUI.
+	 *
+	 * @param task int Task
+	 * @param destination int Destination <code>id</code>
+	 * @return List<TaskResult> Results
+	 */
+	getTaskResultsByTaskAndDestination {
+		public List<TaskResult> call(ApiCaller ac, Deserializer parms) throws PerunException {
+			return ac.getTasksManager().getTaskResultsByTaskAndDestination(ac.getSession(), parms.readInt("task"), parms.readInt("destination"));
+		}
+	},
+
 	/*#
 	 * Return list of only newest TaskResults by a Task for GUI.
 	 *
 	 * @param task int Task
 	 * @return List<TaskResult> Results
 	 */
 	getTaskResultsForGUIByTaskOnlyNewest {
+		@Deprecated
```

Comment:
I think it is also possible to put the `@Deprecated` annotation into the pseudo-javadoc, which is used for the generation of our documentation. I'm not sure if this would work the same way as well.

Reply:
Added, but in javadoc it's @deprecated (with small d).